### PR TITLE
Customer booking func

### PR DESCRIPTION
**Pull request by kimmccaskill**
#### Questions:

### CODE REVIEW

**Comment by kimmccaskill** [comment]
@ZaneMeroff

File: src/css/_guest-layout.scss [hunk 1]
```diff
@@ -142,3 +141,38 @@
...
   margin-top: 10px;
 }
+
+.available-booking-card {
+  font-family: $secondary-font;
+  width: 95%;
+  padding: 5%;
+  border-bottom: black solid 1px;
+  display: flex;
+  flex-direction: row;
```
Comment (ZaneMeroff) [inline src/css/_guest-layout.scss]: I like your use of padding on the bottom boarder in order to make a "partition" between possible bookings.

File: src/css/_guest-layout.scss [hunk 2]
```diff
@@ -142,3 +141,38 @@
   font-family: $secondary-font;
   margin-top: 10px;
 }
+
+.available-booking-card {
+  font-family: $secondary-font;
```
Comment (ZaneMeroff) [inline src/css/_guest-layout.scss]: Good use of SASS to create variables like _$secondary-font_

File: src/index.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ import './css/base.scss';
 import User from './User'
 import Bookings from './Bookings'
 
-let currentUser, bookings, userBookings;
+let currentUser, bookings, userBookings, selectedDate;
 let todaysDate = new Date();
 let dd = String(todaysDate.getDate()).padStart(2, '0');
 let mm = String(todaysDate.getMonth() + 1).padStart(2, '0');
```
Comment (ZaneMeroff) [inline src/index.js]: You may want to eventually make line 10 more dynamic by writing a function only adding a "0" when necessary. I'm not 100% sure I'm reading that correctly though.

File: src/index.js [hunk 4]
```diff
@@ -35,7 +35,7 @@ Promise.all([userData, roomData, bookingData])
     roomData = data[1];
     bookingData = data[2];
   })
-  .then(response => doThing());
+  .then(response => doThing())
 
 const doThing = () => {
```
Comment (ZaneMeroff) [inline src/index.js]: I would recommend refactoring doThing() to a more semantic name.

File: src/index.js [hunk 5]
```diff
@@ -49,16 +49,20 @@ const logIn = () => {
...
+
+export const returnDate = () => {
+  return selectedDate;
+}
+
 const checkCredentials = (id) => {
   if((!id || id > 50) && $(".user-input").val() !== "manager") {
     // Error handling for long username needed
     console.log("error handling for improper credentials")
     return;
```
Comment (ZaneMeroff) [inline src/index.js]: I don't see any actual "error handling" other than the console.log

**Pull request merged**